stable.js

```diff
@@ -0,0 +1,591 @@
+'use strict'
```

Comment:
what is this new file?

Reply:
This is a leftover after rebased to the new main branch